src/modules/auth/auth.controller.ts

```diff
@@ -122,4 +122,28 @@ export class AuthController extends Controller {
       data: "Successfully signedout from all devices",
     });
   }
+  //forgot password function
+  @Post("/forgotPassword")
+  @Middlewares(validateData(authValidations.login.omit({ password: true })))
+  public async forgotPassword(
+    @Body() body: AuthVerifyEmail
+  ): Promise<APIResponse<string>> {
+    try {
+      const user = await userService.fetchByEmail(body.email);
+
+      if (!user) {
+        this.setStatus(errorConst.notFound.code);
+        return toResponse({ error: errorConst.notFound.message });
+      }
+
+      await otpService.send(user);
+
+      return toResponse({
+        data: "Forgot password email send successfully",
+      });
```

Comment:
Once you have updated the response type to `Promise<APIResponse<Message>>`, this should become

```javascript
return toResponse({
  data:  { message: "Forgot password email send successfully" },
});
```